functional/FunctionalTest.java

```diff
@@ -340,6 +340,16 @@ private static void handleException(String methodName, String args, long startTi
     throw e;
   }
 
+  private static void deleteFilesAndObjects(Map<String, List<String>> deleteList) throws Exception {
```

Comment:
The argument doesn't have to be a map as you're always passing a single bucket. I suggest the following signature for this method:

`private static void deleteFilesAndObjects(String bucket, String files[])`

Reply:
Objects are related to bucket. In case we have more one bucket then `private static void deleteFilesAndObjects(String bucket, String files[])` will not be of use. So i have created a map with bucketName as key and object as list of String.

Reply:
This is just a method to reduce verbosity and duplication of code. If this itself makes the code more verbose, then the purpose is defeated.

Since all places where this is called currently work with a single bucket, I don't see a reason to add the complexity of supporting multiple buckets at this point.